Code/Tools/Android/ProjectBuilder/build.gradle.in

```diff
@@ -30,6 +30,7 @@ ${NATIVE_CMAKE_SECTION_DEBUG_CONFIG}
             ${SIGNING_DEBUG_CONFIG}
         }
         profile {
+            getIsDefault().set(true)
```

Comment:
This can be ignored, it's part of another PR https://github.com/o3de/o3de/pull/10191